src/ws/cockpit-session-client-certificate.c

```diff
@@ -121,7 +121,7 @@ sssd_map_certificate (const char *certificate, char** username)
 }
 
 char *
-cockpit_session_client_certificate_map_user (void)
+cockpit_session_client_certificate_map_user (const char *client_certificate_filename)
```

Comment:
Having this API change in this `tls: add a "client-certificate" field to metadata` commit is a little weird, as this functino does not use the new argument in any way. I see that a later commit does, though, so maybe this will just get cleaned up during rebasing.